app/public/js/location_search_table.js

```diff
@@ -0,0 +1,40 @@
+function bindeUpdateRadius() {
+	preventSubmitForm();
+	$("#updateRadius").click(onclickUpdateButtonEvent);
+	function onclickUpdateButtonEvent(event){
+		var updateRadiusApi = "/api/set-search-radius"
+		var distNumber = parseFloat($("#distance").val());
+		if (!distNumber || distNumber > 200  || distNumber<=0 ) {
+			$("dev.disform").addClass("has-error");
+			$("#distance").val("");
+			return;
+		}
+		$.ajax(
+			{
+				url: updateRadiusApi,
```

Comment:
The updateRadiusApi is used only once so it would be better to substitute its value in rather than put it in a variable and use the variable.

Remove the updateRadiusApi variable and do like this:
url: "/api/set-search-radius"